core/src/main/scala/spinal/core/internals/ComponentEmitterVerilog.scala

```diff
@@ -1728,8 +1736,36 @@ end
     }
   }
 
+  var outputSignalNoUse: Set[BaseType] = Set()
```

Comment:
Scala immultable Set do not scale.
You can use the mutable.LinkedHashSet instead